### DISCUSSION TITLE
Será que um dia teremos um revival ou um reboot desse chatbot maravilhoso?

### DISCUSSION THREAD

**Discussion by renanalencar** [GabrielRF/RastreioBot#170] [Geral]
Depois que o chatbot foi encerrado minha vida de rastreio de encomendas enviadas pelos Correios se tornou um pesadelo. Ter que depender daquele app super bugado não é mole. Será que um dia teremos um revival deste chatbot genial criado por @GabrielRF ?

**Comment by GabrielRF** [comment]
@renanalencar Sem dúvida é um desejo meu reviver o bot. Porém, eu somente o faria com o devido acesso ao sistema dos Correios.

**Comment by renanalencar** [comment]
GabrielRF/RastreioBot, lembro que uma discussão alguém mencionou que tem
como ter acesso a nova API através de um cadastro com CNPJ. Parece que o
cadastro e acesso são gratuitos. Não sei se você chegou a verificar isso.

Em qui., 4 de jan. de 2024 às 10:23, Gabriel R F ***@***.***>
escreveu:

> @renanalencar <https://github.com/renanalencar> Sem dúvida é um desejo
> meu reviver o bot. Porém, eu somente o faria com o devido acesso ao sistema
> dos Correios.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GabrielRF/RastreioBot/discussions/170#discussioncomment-8012712>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAF5S7WPEUGIAJYYWFOCSWDYM2UN3AVCNFSM6AAAAABBLKFSO6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAMJSG4YTE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

[image: RCA Desenvolvimento e Consultoria]
<https://htmlsig.com/t/000001BWWD6E>

Renan Costa Alencar
Computer Scientist | PhD Student in Computer Engineering (Computer
Intelligence)
***@***.*** / http://lattes.cnpq.br/3381678604898700
+55 (81) 98233-5881 / +55 (81) 99844-0630

RCA Desenvolvimento e Consultoria
+55 (81) 99844-0630
R José de Holanda, TorreRecife - PE, Brasil CEP 50710-140
http://www.renanalencar.eng.br

**Reply by GabrielRF** [reply]
@renanalencar 
>Parece que o cadastro e acesso são gratuitos.

Sim, é verdade.
Porém, como expliquei no texto linkado no bot, eu não tenho CNPJ e, mesmo que eu faça um, não é garantia que os Correios irão me dar acesso.
São etapas..
1. Abrir um CNPJ;
2. Fazer contrato com os Correios no nível Bronze ou melhor;
3. Pedir acesso à API de rastreio de pacotes.

As etapas 2 e 3 dependem exclusivamente dos Correios, não de mim.

**Reply by Camillaguimaraes** [reply]
Se você tiver o contrato, é garantido o acesso as APIs preço, prazo, rastro e PPN. https://www.correios.com.br/correios-facil/correios-facil
